### ISSUE TITLE
*CRITICAL ISSUE* Frontend: When using turbulent wind type, the request sent to backend does not contain any related information

### ISSUE THREAD

**Issue by BohanZhang1** [oss-slu/DroneWorld#89]
Configuration:
![image](https://github.com/oss-slu/DroneWorld/assets/113057382/43a4fdcf-d5aa-4bc7-b817-e866a1776d9d)

Raw request, environment field:
```
"environment": {
        "Wind": {
            "Direction": "N",
            "Velocity": 1,
            "Force": 10
        },
        "Origin": {
            "Latitude": 41.980381,
            "Longitude": -87.934524
        },
        "TimeOfDay": "10:00:00",
        "UseGeo": true
    }
```

**Comment by BohanZhang1** [comment]
expected request:
```
"environment": {
        "Wind": {
            "Type": "Turbulent",
            "Direction": "N",
            "Force": 10,
            "Fluctuation": 0.9
        },
        "Origin": {
            "Latitude": 41.980381,
            "Longitude": -87.934524
        },
        "TimeOfDay": "10:00:00",
        "UseGeo": true
    }
```

**Comment by BohanZhang1** [comment]
Similar error when using wind shear
![image](https://github.com/oss-slu/DroneWorld/assets/113057382/45894da5-950d-42bb-a65b-7714d4c5395e)

Raw request:
```
"environment": {
        "Wind": {
            "Direction": "N",
            "Velocity": 1,
            "Force": 10
        },
        "Origin": {
            "Latitude": 41.980381,
            "Longitude": -87.934524
        },
        "TimeOfDay": "10:00:00",
        "UseGeo": true
    }
```

Expected:
```
"environment": {
        "Wind": {
            "Wind1":
            {
            "Type": "Turbulent",
            "Direction": "N",
            "Force": 10,
            "Fluctuation": 0.2
            },
            "Wind2":
            {
            "Type": "Turbulent",
            "Direction": "E",
            "Force": 20,
            "Fluctuation": 0.8
            }
        },
        "Origin": {
            "Latitude": 41.980381,
            "Longitude": -87.934524
        },
        "TimeOfDay": "10:00:00",
        "UseGeo": true
    }
```

**Comment by BohanZhang1** [comment]
Fixed in PR #90 
Thanks you Kaleb for the timely response
